src/createReactQueryClient.js

```diff
@@ -0,0 +1,16 @@
+import { QueryClient } from 'react-query';
+
+const cacheTimeMinutes = 5;
+
+const createReactQueryClient = () => new QueryClient({
+  // https://react-query.tanstack.com/guides/important-defaults
+  defaultOptions: {
+    queries: {
```

Comment:
It is possible to set defaults for the mutations as well, but I did not found use cases when the default ones do not fit folio needs. It is important that by default errored mutations won't be retried.